### PR TITLE
br: remove cloud admin filter from br restoration and set --with-sys-table as default value

### PR DESCRIPTION
**Pull request by benmaoer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-snapshot-guide.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ tiup br restore full --pd "${PD_IP}:2379" \
 
 当恢复系统权限相关数据的时候，请注意：
 
-- BR 不会恢复 `user` 为 `cloud_admin` 并且 `host` 为 `'%'` 的用户数据，该用户是 TiDB Cloud 预留用户。请不要在你的环境中创建 `cloud_admin` 的用户或者角色，因为依赖 `cloud_admin` 的用户的权限将不能被完整恢复。
+- `br` v7.6.0 版本之前无法恢复 `user` 为 `cloud_admin` 并且 `host` 为 `'%'` 的用户数据，该用户是 TiDB Cloud 预留用户。`br` v7.6.0 版本开始，默认支持包括  `cloud_admin` 在内的所有用户恢复。
```
Comment (Oreoxmt) [inline br/br-snapshot-guide.md:170]:
```suggestion
- 在 v7.6.0 之前版本中，BR 无法恢复 `user` 为 `cloud_admin` 并且 `host` 为 `'%'` 的用户数据，该用户是 TiDB Cloud 预留用户。从 v7.6.0 开始，BR 默认支持恢复包括 `cloud_admin` 在内的所有用户数据。
```

File: br/br-snapshot-guide.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ tiup br restore full --pd "${PD_IP}:2379" \
 
 ### 恢复 `mysql` 数据库下的表
 
-自 `br` v5.1.0 开始，快照备份会备份 **mysql schema 下的系统表数据**，而不会默认恢复这些数据。自 `br` v6.2.0 开始，在设置 `--with-sys-table` 下，恢复数据时将同时恢复**部分系统表相关数据**。
+`br` v5.1.0 开始，快照备份时默认自动备份 **mysql schema 下的系统表数据**，但不默认恢复系统表数据。`br` v6.2.0 开始，增加恢复参数 `--with-sys-table` 支持恢复数据的同时恢复**部分系统表相关数据**。 `br` v7.6.0 开始，恢复参数 `--with-sys-table`  默认打开，默认支持恢复数据的同时恢复**部分系统表相关数据**。
```
Comment (Oreoxmt) [inline br/br-snapshot-guide.md:122]:
```suggestion
`br` v5.1.0 开始，快照备份时默认自动备份 **mysql schema 下的系统表数据**，但不默认恢复系统表数据。`br` v6.2.0 开始，增加恢复参数 `--with-sys-table` 支持恢复数据的同时恢复**部分系统表相关数据**。`br` v7.6.0 开始，恢复参数 `--with-sys-table` 默认开启，即默认支持恢复数据的同时恢复**部分系统表相关数据**。
```

Comment (Oreoxmt) [reply br/br-snapshot-guide.md:122]: `br-snapshot-manual.md` 文档中是否需要添加 `--with-sys-table` 从 7.6 开始默认开启的说明？

https://github.com/pingcap/docs-cn/blob/2ef79fcb75464e5e48ba1e2744ee66cb0ecceeb4/br/br-snapshot-manual.md?plain=1#L165-L182

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

File: br/br-snapshot-guide.md [hunk 3]
```diff
@@ -119,7 +119,7 @@ tiup br restore full --pd "${PD_IP}:2379" \
 
 ### 恢复 `mysql` 数据库下的表
 
-自 `br` v5.1.0 开始，快照备份会备份 **mysql schema 下的系统表数据**，而不会默认恢复这些数据。自 `br` v6.2.0 开始，在设置 `--with-sys-table` 下，恢复数据时将同时恢复**部分系统表相关数据**。
+`br` v5.1.0 开始，快照备份时默认自动备份 **mysql schema 下的系统表数据**，但不默认恢复系统表数据。`br` v6.2.0 开始，增加恢复参数 `--with-sys-table` 支持恢复数据的同时恢复**部分系统表相关数据**。`br` v7.6.0 开始，恢复参数 `--with-sys-table` 默认开启，即默认支持恢复数据的同时恢复**部分系统表相关数据**。
```
Comment (qiancai) [inline br/br-snapshot-guide.md:122]:
```suggestion
- `br` v5.1.0 开始，快照备份时默认自动备份 **mysql schema 下的系统表数据**，但恢复数据时默认不恢复系统表数据。
- `br` v6.2.0 开始，增加恢复参数 `--with-sys-table` 支持恢复数据的同时恢复**部分系统表相关数据**。
- `br` v7.6.0 开始，恢复参数 `--with-sys-table` 默认开启，即默认支持恢复数据的同时恢复**部分系统表相关数据**。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**